test/irb/test_color.rb

```diff
@@ -155,6 +155,18 @@ def test_colorize_code
       end
     end
 
+    def test_colorize_code_with_local_variables
+      code = "a /(b +1)/i"
+      result_without_lvars = "a #{RED}#{BOLD}/#{CLEAR}#{RED}(b +1)#{CLEAR}#{RED}#{BOLD}/i#{CLEAR}"
+      result_with_lvar = "a /(b #{BLUE}#{BOLD}+1#{CLEAR})/i"
+      result_with_lvars = "a /(b +#{BLUE}#{BOLD}1#{CLEAR})/i"
+      if colorize_code_supported?
```

Comment:
This is not necessary because irb requires Ruby 2.5+, which always satisfy this condition. I added https://github.com/ruby/irb/pull/413 to remove this obsolete check.